server/routes/approvedVisitors.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+
+const createApprovedVisitorsRouter = ({ offenderService }) => {
+  const router = express.Router();
+
+  const getPersonalisation = async (user, query) => {
+    const rawApprovedVisitors = await offenderService.getApprovedVisitorsFor(
+      user,
+    );
+    const { page: rawPage = 1 } = query;
+    const maxVisitorsPerPage = 10;
+    const totalCount = rawApprovedVisitors.length;
+    const page =
+      Number.isNaN(rawPage) || rawPage > totalCount ? 1 : Number(rawPage);
+    const totalPages = Math.ceil(totalCount / maxVisitorsPerPage);
+    const max = page < totalPages ? page * maxVisitorsPerPage : totalCount;
+    const min = page === 1 ? 1 : (page - 1) * maxVisitorsPerPage + 1;
```

Comment:
Meh.. not sure what happens if you pass in `page < 0`

Would using Math.min be more appropriate?